### PR TITLE
fix system tables field datatype

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fix: `SELECT * from system.processes` failed when serialize_json in http hander

## Changelog

- Bug Fix
## Related Issues

Fixes  issue https://github.com/datafuselabs/databend/issues/4302
## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: query/src/storages/system/processes_table.rs [hunk 1]
```diff
@@ -105,7 +105,7 @@ impl ProcessesTable {
             DataField::new("state", Vu8::to_data_type()),
             DataField::new("database", Vu8::to_data_type()),
             DataField::new_nullable("extra_info", Vu8::to_data_type()),
-            DataField::new_nullable("memory_usage", i64::to_data_type()),
+            DataField::new("memory_usage", i64::to_data_type()),
```
Comment (youngsofun) [inline query/src/storages/system/processes_table.rs:108]: data is i64 not Option<i64>

**Comment by BohuTANG** [comment]
Which issue does this PR addressed? We can set it to `Fixes #<issue-no>`

**Comment by BohuTANG** [comment]
Is it possible to add a http handler unit test for `SELECT * from system.processes`?

**Comment by youngsofun** [comment]
@BohuTANG ok

**Comment by youngsofun** [comment]
@BohuTANG fixed 2 similar bugs.

File: query/src/interpreters/interpreter_query_log.rs [hunk 2]
```diff
@@ -116,7 +116,7 @@ impl InterpreterQueryLog {
             Series::from_data(vec![event.query_kind.as_str()]),
             Series::from_data(vec![event.query_text.as_str()]),
             Series::from_data(vec![event.event_date as i32]),
-            Series::from_data(vec![event.event_time as u64]),
+            Series::from_data(vec![event.event_time as i64]),
```
Comment (youngsofun) [inline query/src/interpreters/interpreter_query_log.rs:119]: event.event_time is u64, but query_load table field type DateTime64Type maps to i64, is this safe?

**Comment by youngsofun** [comment]
Warning: error: 1 vulnerability found!

**Comment by BohuTANG** [comment]
@mergify update

**Review by BohuTANG — APPROVED** [review]
Looks good!

**Comment by youngsofun** [comment]
rebase only amend the last commit

**Comment by youngsofun** [comment]
test_stateless_cluster

Error: Process completed with exit code 1.

**Comment by youngsofun** [comment]
@BohuTANG can I rerun fail tests by myself such as https://github.com/datafuselabs/databend/runs/5432388984?check_suite_focus=true

File: query/src/interpreters/interpreter_query_log.rs (hunk 2 above)
Comment (BohuTANG) [reply query/src/interpreters/interpreter_query_log.rs:119]: Why we change it to i64 here?

Comment (BohuTANG) [reply query/src/interpreters/interpreter_query_log.rs:119]: Ok, I see. I think it's ok here

**Pull request merged**